Shopify/starter-theme
# refactor login.js and login.liquid to remove jquery dependency

This PR closes #138 
Inspired by the excellent work done on #122 

### Changes
- [x] update selectors in `login.liquid` to use data-attributes
- [x]  update `login.js` to reference new selectors and replace jquery with vanilla js
  
### How to test

- Preview with https://smkktest123.myshopify.com
- Make a user account
- Check you can login as intended
- Check you can trigger password reset and a confirmation message is shown

### Expectations:

- [x] It functions as before
- [x] It follows the same code style that we are using on `addresses.js`
